### ISSUE TITLE
Intsallation issue when Docker is installed (but not used by Pihole)

### ISSUE THREAD

**Issue by TiuTalk** [jacklul/pihole-updatelists#82]
I walked through the `install.sh` script, and this line:

```
DOCKER=`grep -sq 'docker\|lxc' /proc/1/cgroup && echo "0" || echo "1"` # Is this a Docker installation?
```

will try to do use Docker to setup updatelists and fail, see logs below:

```
$ wget -O - https://raw.githubusercontent.com/jacklul/pihole-updatelists/master/install.sh | sudo bash
--2021-08-12 15:44:20--  https://raw.githubusercontent.com/jacklul/pihole-updatelists/master/install.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.111.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 5885 (5.7K) [text/plain]
Saving to: ‘STDOUT’

-                                100%[========================================================>]   5.75K  --.-KB/s    in 0.001s

2021-08-12 15:44:21 (6.11 MB/s) - written to stdout [5885/5885]

2021-08-12 15:44:22 URL:https://raw.githubusercontent.com/jacklul/pihole-updatelists/master/pihole-updatelists.php [70445/70445] -> "/usr/local/sbin/pihole-updatelists" [1]
mode of '/usr/local/sbin/pihole-updatelists' changed from 0644 (rw-r--r--) to 0755 (rwxr-xr-x)
2021-08-12 15:44:22 URL:https://raw.githubusercontent.com/jacklul/pihole-updatelists/master/pihole-updatelists.bash [1141/1141] -> "/etc/bash_completion.d/pihole-updatelists" [1]
bash: line 168: /etc/cont-init.d/10-pihole-updatelists.sh: No such file or directory
```

I had to edit the install.sh script and force `DOCKER=0` for the install to work

**Comment by jacklul** [comment]
Deployed test fix to dev branch, please try installing from it directly, I'm currently unable to test the fix myself.

**Comment by TiuTalk** [comment]
Looks correct! Thanks @jacklul 

Closed by e2a62b3d081c4
